pages/phases/job-analysis/creating-assessments-from-competencies.md

```diff
@@ -1,19 +1,31 @@
 ---
 layout: article
-permalink: /hiring-phases/job-analysis/creating-a-structured-interview-from-competencies/
+permalink: /hiring-phases/job-analysis/creating-assessments-from-competencies/
 section: hiring-phases
-title: Creating a Structured Interview from Competencies
+title: Creating Assessments from Competencies
 sidenav: hiring-phases
 phase: job-analysis
 sub-phase-order: 060
-intro: Finish your workshop by breaking participants into groups to write questions for both the first and second phone assessment interviews. Use your defined competencies, proficiencies, and required experience to create two rounds of interview questions and good example responses.
+intro: Finish your workshop by breaking participants into groups to write questions for both the first and second assessment rounds. Use your defined competencies, proficiencies, and required experience to create guidance for SMEs reviewing applicant responses.
```

Comment:
"finish your workshop by" sounds like the very end vs the entire second half. Maybe "Dedicate the second half of your job analysis workshop to working with SMEs to develop the material for both assessment rounds. The SMEs should decide which assessment types make sense to evaluate each competency and then create assessments (interview questions and/or written prompts" to assess the identified proficiency levesls."